app/javascript/mastodon/features/navigation_panel/components/followed_tags_panel.tsx

```diff
@@ -54,16 +57,26 @@ export const FollowedTagsPanel: React.FC = () => {
       expandTitle={intl.formatMessage(messages.expand)}
       loading={loading}
     >
-      {tags.map((tag) => (
+      {tags.slice(0, TAG_LIMIT).map((tag) => (
         <ColumnLink
+          transparent
           icon='hashtag'
           key={tag.name}
           iconComponent={TagIcon}
           text={`#${tag.name}`}
           to={`/tags/${tag.name}`}
-          transparent
         />
       ))}
+      {hasMoreTags && (
```

Comment:
I would be more at peace if we didn't have a "View all" link. The top link already links to all tags. I'd like to keep the menus as short as possible.

Reply:
I'd feel bad about closing it, it's a feature request, and it has merit, we just need to invest way more thinking into a proper solution now than makes sense for 4.4.